boards/arc/hsdk4xd/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+#
+# Copyright (c) 2022 Synopsys, Inc. All rights reserved.
```

Comment:
Minor issue: here and in other plases we use 2022 instead of 2023

Reply:
This is marked as resolved, though 2022 year is still used in many places here.

Reply:
From previous comment I understood it like "We SHOULD use 22 instead of 23". So I turned all dates back to 2022 =)